D3D11Engine/GothicMemoryLocations.h

```diff
@@ -27,11 +27,12 @@ static void PatchAddr( unsigned int adr, const T( &v )[n] ) {
 }
 
 // -- call macro from GothicX (thx, Zerxes!)
-#define XCALL(uAddr)                    \
-        __asm { mov esp, ebp    }       \
-        __asm { pop ebp                 }       \
-        __asm { mov eax, uAddr  }       \
-        __asm { jmp eax                 }
+#define XCALL(address)      \
```

Comment:
Why was this changed?

Reply:
MSVC would warn in `pop ebp` that modifying the EBP register is not safe, also `leave` is an optimization over the hand-coded `mov esp, ebp; pop ebp`.